lib/src/main/java/org/team100/lib/localization/NotePosition24ArrayListener.java

```diff
@@ -16,14 +20,16 @@
 public class NotePosition24ArrayListener {
 
     StructBuffer<NotePosition24> m_buf = StructBuffer.create(NotePosition24.struct);
-    NotePosition24[] positions = new NotePosition24[0];
+    NotePosition24[] positions;
+    private String id;
```

Comment:
why is this a field of this class?

Reply:
Would it be better to have it as a CameraAngles field and just use the ID to feed the parems?

Reply:
i think you are imagining a specific sequence of method calls in this class, but nothing guarantees that sequence.  the "consumeValues" method can be called at any time by a dedicated networktables listener thread.  the other methods can be called at any time by a different thread.  you should do *all* the work of calculation in the consumeValues method, and the other methods should just retrieve the results atomically.